dev/landing-pages/this-is-big.scss

```diff
@@ -0,0 +1,1336 @@
+@import '../media-template/media-template-base';
+
+$header-media-xs-url:   '//www.ucf.edu/wp-content/uploads/2017/08/this-is-big-graduation-1-1.jpg';
+$section-quote-bg-md:   '//www.ucf.edu/wp-content/uploads/2017/08/this-is-big-teacher-3.jpg';
+
+$gray: #ececec;
+$dark-gray: #242424;
+
+#header-menu {
+  li:not(:last-child) a,
+  li:not(.last) a {
+    color: $white;
+
+    &:hover {
+      color: $ucf-gold;
+    }
+  }
+}
+
+.page-media-container {
+  bottom: 0;
+  top: 0;
+}
+
+.page-media {
+  @media (max-width: $screen-sm-max) {
+    margin-bottom: -40px;
+    top: -70px;
+    position: relative;
+  }
+  @media (max-width: $screen-xs-max) {
+    margin-bottom: 0;
+    top: -10px;
+    position: relative;
+  }
+
+  .page-media-header {
+    background-position: top center;
+    border-bottom: 3px solid $ucf-gold;
+    color: #fff;
```

Comment:
Not a big deal, but you could use the $white and $black variables in a couple places.